.travis.yml

```diff
@@ -36,7 +36,7 @@ before_install:
 install:
   - travis_retry npm install -g grunt-cli
   - travis_retry npm install
-  - travis_retry pip install -r requirements.txt --download-cache $HOME/.pip-cache
+  - travis_retry pip install -r config/requirements.txt --download-cache $HOME/.pip-cache
```

Comment:
We had a PR update the Travis.yml file to not use this manual `--download-cache` thing, so there's probably a conflict here. Let me know if you need help resolving.